### DISCUSSION TITLE
Nomad Network compact announce stream

### DISCUSSION THREAD

**Discussion by markqvist** [markqvist/Reticulum#364] [Show and tell]
Just wanted to say that there's now a "compact announce stream" option in Nomad Network version `0.3.6`. Enable it in the `[client]` section of the config:

```
[client]
compact_announce_stream = yes
```

When enabled, only one entry will be shown for each announced destination, even someone is announcing often.

**Comment by faragher** [comment]
Is this Echobot's fault? Sorry. I just figured it would save a lot of unreplied to "test" messages I keep getting on various endpoints.

**Reply by markqvist** [reply]
Not at all! I just wanted that feature for myself for a while now, since it makes it easier to quickly spot any new nodes when you don't check the announce stream so often.
